net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -63,9 +63,11 @@ function arpwatch_install_command() {
 			    .' '.$disable_zero
 			    .' '.$enable_zeropad
 			    .' -f '.escapeshellarg($arp_file)
-			    .' -i '.escapeshellarg($ifname)
-			    .' -w '.escapeshellarg($notifications_recipient)
-			    ."\n";
+			    .' -i '.escapeshellarg($ifname);
+			    if (!empty($notifications_recipient)) {
+				    $rc['start'] .= ' -w '.escapeshellarg($notifications_recipient);
+			    }
+			    $rc['start'] .= "\n";
```

Comment:
These lines do not need to be indented the extra spaces here. They should be aligned with line 60. Lines 61-66 are indented more as they are continuations of the statement on line 60.

Reply:
Fixed